### PR TITLE
Created makefile and example.mk for ES secrets

### PR DESCRIPTION
**Pull request by somanath21**
## Description

* Created a Folder for ES Security to create ES-secrets using makefile and example.mk file 
* mention namespace in a makefile to create ES secrets in specific namespaces.

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/ES-security/Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	docker run --name elastic-helm-charts-certs -i -w /app \
+		$(ELASTICSEARCH_IMAGE) \
+		/bin/sh -c " \
+			elasticsearch-certutil ca --out /app/elastic-stack-ca.p12 --pass '' && \
+			elasticsearch-certutil cert --name security-master --dns security-master --ca /app/elastic-stack-ca.p12 --pass '' --ca-pass '' --out /app/elastic-certificates.p12" && \
+	docker cp elastic-helm-charts-certs:/app/elastic-certificates.p12 ./ && \
+	docker rm -f elastic-helm-charts-certs && \
+	openssl pkcs12 -nodes -passin pass:'' -in elastic-certificates.p12 -out elastic-certificate.pem && \
+	openssl x509 -outform der -in elastic-certificate.pem -out elastic-certificate.crt && \
+	kubectl create namespace paymenthub || echo "namespace already exists"  && \
```
Comment (fynmanoj) [inline helm/ES-security/Makefile:28]: can we externalize these namespace variables or make it as a global variable and set it only once?

File: helm/kibana-secret/makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+test: secrets install goss
+
+purge:
+	kubectl delete secret kibana || true
+	helm del $(RELEASE)
+
+secrets:
+	encryptionkey=$$(docker run --rm busybox:1.31.1 /bin/sh -c "< /dev/urandom tr -dc _A-Za-z0-9 | head -c50") && \
+  kubectl create namespace paymenthub || echo "namespace already exists"  && \
```
Comment (fynmanoj) [inline helm/kibana-secret/makefile:16]: can we externalize these namespace variables or make it as a global variable and set it only once?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
changes requested

File: helm/ES-security/Makefile (hunk 1 above)
Comment (somanath21) [reply helm/ES-security/Makefile:28]: This line works for creating secrets in specific NS. Just need to change the <namespace> in makefile before running the pipeline.

File: helm/kibana-secret/makefile (hunk 2 above)
Comment (somanath21) [reply helm/kibana-secret/makefile:16]: This line works for creating secrets in specific NS. Just need to change the <namespace> in makefile before running the pipeline.

**Pull request merged**